lib/msf/base/sessions/mysql.rb

```diff
@@ -0,0 +1,142 @@
+# -*- coding: binary -*-
+
+require 'rex/post/mysql'
+
+class Msf::Sessions::MYSQL
+  #
+  # This interface supports basic interaction.
+  #
+  include Msf::Session::Basic
+  include Msf::Sessions::Scriptable
+
+  # @return [Rex::Post::MySQL::Ui::Console] The interactive console
+  attr_accessor :console
+  # @return [MySQL::Client]
+  attr_accessor :client
+  attr_accessor :platform, :arch
+
+  # @param[Rex::IO::Stream] rstream
+  # @param [Hash] opts
+  def initialize(rstream, opts = {})
+    @client = opts.fetch(:client)
+    self.console = ::Rex::Post::MySQL::Ui::Console.new(self)
+    super(rstream, opts)
+  end
+
+  def bootstrap(datastore = {}, handler = nil)
+    session = self
+    session.init_ui(user_input, user_output)
+
+    @info = "MySQL #{datastore['USERNAME']} @ #{client.socket.peerinfo}"
+  end
+
+  def process_autoruns(datastore)
+    ['InitialAutoRunScript', 'AutoRunScript'].each do |key|
+      next if datastore[key].nil? || datastore[key].empty?
+
+      args = Shellwords.shellwords(datastore[key])
+      print_status("Session ID #{session.sid} (#{session.tunnel_to_s}) processing #{key} '#{datastore[key]}'")
+      session.execute_script(args.shift, *args)
```

Comment:
```suggestion
      self.execute_script(args.shift, *args)
```

Potentially needs changed.